perception/traffic_light_classifier/src/nodelet.cpp

```diff
@@ -23,6 +23,13 @@ namespace traffic_light
 TrafficLightClassifierNodelet::TrafficLightClassifierNodelet(const rclcpp::NodeOptions & options)
 : Node("traffic_light_classifier_node", options)
 {
+  auto node_name = this->get_name();
+  if (strcmp(node_name, "traffic_light_classifier_car") == 0) {
+    classify_tl_type_ = tier4_perception_msgs::msg::TrafficLightRoi::CAR_TL;
+  } else if (strcmp(node_name, "traffic_light_classifier_ped") == 0) {
+    classify_tl_type_ = tier4_perception_msgs::msg::TrafficLightRoi::PEDESTRIAN_TL;
+  }
+
```

Comment:
It would be better to switch type by parameter not node_name.

Reply:
https://github.com/autowarefoundation/autoware.universe/pull/5871/commits/d71cd81677b20f2e355ab8a4583b70e4f1afe3e2 It is now switched by launch paramter.